### DISCUSSION TITLE
Flipping protection for Crazyflie2.1

### DISCUSSION THREAD

**Discussion by 7enHom** [bitcraze/discussions#1129] [Q&A] [answered]
Hi！

I'm working on a flip task. But perhaps due to the newer firmware, there is a flip protection written in the firmware that causes the drone to power down due to the protection mechanism when flipped over a large angle, which is not present if an older version of the firmware is installed. I would like to know where in the firmware such protection is written? Can I change this setting in the new version of the firmware?

Thanks a bunch for the info!

**Comment by tobbeanton** [comment, marked as answer]
This has changed a bit with the new supervisor implementation. You can turn off the tumble check by defining [SUPERVISOR_TUMBLE_CHECK_ENABLE](https://github.com/bitcraze/crazyflie-firmware/blob/master/src/platform/interface/platform_defaults.h#L114) to false.

**Reply by 7enHom** [reply]
thanks！
